axelrod/strategies/adaptive.py

```diff
@@ -0,0 +1,49 @@
+from axelrod import Actions, Player, Game
+
+C, D = Actions.C, Actions.D
+
+
+class Adaptive(Player):
+    """Start with a specific sequence of C and D, then play the strategy that
+    has worked best, recalculated each turn."""
+
+    name = 'Adaptive'
+    classifier = {
+        'memory_depth': float('inf'),  # Long memory
+        'stochastic': False,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def __init__(self):
+        Player.__init__(self)
+        self.initial_plays = [C] * 6 + [D] * 5
+        self.reset()
+
+    def score_last_round(self, opponent):
+        try:
+            game = self.match_attributes["game"]
+        except AttributeError:
```

Comment:
Could you add a docstring for this explaining that this uses the game attribute if it is known by the player (neat to have that).